New package: SphereUDE v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: SphereUDE
- Repository: https://github.com/facusapienza21/SphereUDE.jl
- Created by: @facusapienza21
- Version: v0.1.0
- Commit: 9ae62eeeeb1df71094e705468868c2193bbbfb11
- Reviewed by: @facusapienza21
- Reference: https://github.com/facusapienza21/SphereUDE.jl/commit/9ae62eeeeb1df71094e705468868c2193bbbfb11#commitcomment-137089135
- Description: A Julia package for regression in the sphere based on Universal Differential Equations
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6f37222394bfbbbdaedbca1f94ee4c95b46eb8bd8b4467b14c06117b864e9c8fa73a2d746e0108537a7a8527668103cb4cafd4f2af6427d430935f6bbf66637eb494ee6a6e40dfa9479f6ecc58652deeb19b3073a08d02bde927d762490f22d5f1282a333fdba8f16b67ab96bee2de93937c9566977880b882cfc615f366498a8bae2e7de020e6d3bc9661d080f7197976d5bcb8b1c8b14ce3c557b42667741ba810d34c363364c66829508367cd80a44 -->

goerz commented:
Thank you for contributing your package! It would appear that the README isn’t really related to the package itself. Please make sure to add some documentation before registering. Specifically, the README should explain the purpose of the package and give a basic usage example.

facusapienza21 commented:
Thank you @goerz for the comment! I have included relevant information of the package in the `README.md` and also included the required dependencies in the `compat`. I think right now automerge is not possible since some of the dependencies of `SphereUDE.jl` are Python, which is an extra installation steps that is needed to complete by the user (as explained in the readme). Do you think we can still merge in the Registry? Thanks!

[noblock]

goerz commented:
[noblock]

> Thank you @goerz for the comment! I have included relevant information of the package in the `README.md`

I'm not seeing an example for how to do a specific "non-parametric regression of data supported in the three-dimensional sphere". With the current README/documentation I would have no idea how to actually use the package, apart from installing/initializing it.

> I think right now automerge is not possible since some of the dependencies of `SphereUDE.jl` are Python, which is an extra installation steps that is needed to complete by the user (as explained in the readme). 

See https://github.com/JuliaRegistries/General#my-package-fails-to-load-because-it-needs-proprietary-softwareadditional-setup-to-work-what-can-i-do

JuliaRegistrator committed:
New package: SphereUDE v0.1.0

UUID: d7416ba7-148a-4110-b27d-9087fcebab2d
Repo: https://github.com/facusapienza21/SphereUDE.jl.git
Tree: 76965feea3f4822021d09826bf594cb7e08ea5dc

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
